package.json

```diff
@@ -19,6 +19,8 @@
     "@testing-library/jest-dom": "5.16.4",
     "@testing-library/react": "12.1.5",
     "@testing-library/react-hooks": "8.0.0",
+    "@types/jest": "27.4.1",
+    "@types/node": "17.x",
```

Comment:
17.x ?

Reply:
We're using node 17 in the ci, should I pin it to a more specific version?

Reply:
Latest version should be backward compatible no ?